kernel/configs/fragments/arm64/base.conf

```diff
@@ -0,0 +1,42 @@
+CONFIG_ARM64=y
+CONFIG_ARM64_4K_PAGES=y
+
+# ARM servers are often multi-cores, following configs improve
+# the CPU scheduler's decision making.
+CONFIG_SCHED_MC=y
+CONFIG_SCHED_SMT=y
+
+# Virtual address space size (48-bit)
+CONFIG_ARM64_VA_BITS_48=y
+CONFIG_ARM64_VA_BITS=48
+# Physical address space size (48-bit)
+CONFIG_ARM64_PA_BITS_48=y
+CONFIG_ARM64_PA_BITS=48
+
+# Use the maximum number of CPUs supported by KVM (255)
+CONFIG_NR_CPUS=255
+
+CONFIG_PERF_EVENTS=y
```

Comment:
just to note, iirc, we are debating if we need perf on/off by default in the kata kernels I think - iirc, they may come with a startup perf cost?

Reply:
IMO, we could make perf off by default. 
But for x86_64, this config `CONFIG_PERF_EVENTS` is automatically selected by `CONFIG_KVM`,  See https://github.com/torvalds/linux/blob/master/arch/x86/kvm/Kconfig#L41.
And we select `CONFIG_CGROUP_PERF=y` in common cgroup config file, which is dependent on `CONFIG_PERF_EVENTS`.
I'll try to dump kernel start up info to track the cost. ;)

Reply:
Oh, that's interesting - didn't know KVM selected PERF_EVENTS. I wonder if they are needed for something specific - would you know @dagrh ?

I guess having them on may do no harm - but, I also guess that is now an architecture specific decision at some level - so probably your choice for this arch @Pennyzct  :-)

Reply:
That also surprises me; my reading of it is that it's used to emulate a PMU in the guest on x86 - although I thought the hardware PMU helped with that these days,